job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/ScmContext/perforcep4.groovy

```diff
@@ -0,0 +1,18 @@
+job('example-1') {
+    scm {
+        perforcep4('p4_credentials') {
+            manual('ws_name', '//depot/Tools/build //ws_name/build')
```

Comment:
The view specification here should have /... on the end. e.g.
`            manual('ws_name', '//depot/Tools/build/... //ws_name/build/...')`